util/topgen/templates/toplevel_racl_pkg.sv.tpl

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 ${gencmd}
+<% import textwrap %>\
 <% racl_role_vec_len = 2 ** racl_config['nr_role_bits'] %>\
 
 package top_racl_pkg;
```

Comment:
Weird this doesn't get some uniquifying name. The name top_racl_pkg can become confusing in multi-top.